rhesus-monkeys/scripts/convert-meta.py

```diff
@@ -0,0 +1,89 @@
+import polars as pl
+import re
+from pathlib import Path
+import janitor.polars
+
+# Set the folder path for raw-data
+resource_dir = Path(__file__).resolve().parent.parent
+folder_path = resource_dir / "data-raw"
+unzip_path = folder_path / "unzipped"
+
+# Create the infant metadata files
+    # FIO All infants in meta_infant_ur are included in meta_infant_bl. There is one extra infant in meta_infant_bl.
+df_meta_infant_bl = pl.read_csv(unzip_path / "metadata_infant_blood.csv") 
+df_meta_infant_ur = pl.read_csv(unzip_path / "metadata_infant_urine.csv") 
+
+df_infant = df_meta_infant_bl.select(["Infant_ID", "Mother_ID", "GD_Delivery", "Fostered"
+, "Foster_ID"]).rename({"GD_Delivery": "gestation_day_delivery"}).unique().clean_names().write_csv(folder_path / "data_infant_meta.csv", separator=";")
+    # infant gender is in data_adult at present, all subjects are male, females are not included in the study
+
+df_infant_w1 = df_meta_infant_bl.select(["Infant_ID", "Weight_PD7", "ActualDay_PD7"]).rename({"Weight_PD7": "weight_at_x_days_old", "ActualDay_PD7": "x_days_old"}).clean_names()
+df_infant_w2 = df_meta_infant_ur.select(["Infant_ID","Infant_weight","PD"]).rename({"Infant_weight": "weight_at_x_days_old", "PD": "x_days_old"}).clean_names()
+df_infant_weight = pl.concat([df_infant_w1, df_infant_w2]).unique().write_csv(folder_path / "data_infant_weight.csv", separator=";") # Concatenate (long)
```

Comment:
Use `unpivot()` rather than creating two new datasets and using `concat()`.

Also use method chaining, where each new chain is a new line. Rather than have it all on one line.